react-hooks/intro-to-react-hooks/src/exercises/exercise2/solutions/part1Solution.js

```diff
@@ -15,15 +15,15 @@ const NumberTrivia = (props) => {
   );
   // 📗 Solution below 📗
   // We have also provided the second parameter to
-  // useEffect, an array of dependencies. We only
+  // useEffect, an array of values. We only
   // want to re-run this effect if the number prop
   // changes, otherwise the hook will run every time
   // the component re-renders (setTrivia changes the
   // state of the component, which triggers a re-render!)
 
   // See what happens in the editor if you pass an empty
   // array. You should see an ESLint warning, telling
-  // you that there's a missing dependency (number).
+  // you that there's a missing value (number).
```

Comment:
Same as above, we could change this to something like 
```suggestion
  // you that there's a missing value (number) in the dependency array.
```